### PR TITLE
TC_04.001.14 | Folder> Rename Folder > Verify the user can rename a Folder

### PR DESCRIPTION
**Pull request by AramH20**
https://trello.com/c/NJ2AuaRR

### CODE REVIEW

**Review by verafes — COMMENTED** [review]
Please add this PR link to your Trello card

**Comment by AramH20** [comment]
add imports

File: src/test/java/school/redrover/CreateFolder1Test.java [hunk 1]
```diff
@@ -40,4 +43,29 @@ public void testNewlyCreatedFolderIsEmptyAJ() {
         Assert.assertEquals(actualNewJobLinkText, createAJobLinkText);
         Assert.assertTrue(newJobLink.isDisplayed(), "newJobLink is NOT displayed");
     }
+
+    @Test
+    public void testRenameFolder() throws InterruptedException {
```
Comment (verafes) [inline src/test/java/school/redrover/CreateFolder1Test.java:48]: Please remove InterruptedException

File: src/test/java/school/redrover/CreateFolder1Test.java [hunk 2]
```diff
@@ -40,4 +43,29 @@ public void testNewlyCreatedFolderIsEmptyAJ() {
...
+        final String folderName = "ProjectFolder";
+        final String newFolderName = "NewProjectFolder";
+        final String thisFolderIsEmptyMessage = "This folder is empty";
+        final String createAJobLinkText = "Create a job";
```
Comment (verafes) [inline src/test/java/school/redrover/CreateFolder1Test.java:52]: The variavles thisFolderIsEmptyMessage and thisFolderIsEmptyMessage are unused in the test.

File: src/test/java/school/redrover/CreateFolder1Test.java [hunk 3]
```diff
@@ -40,4 +43,29 @@ public void testNewlyCreatedFolderIsEmptyAJ() {
...
+
+        createNewFolder(folderName);
+        openDashboard();
+
+        WebElement projectName = getWait5().until(ExpectedConditions.visibilityOfElementLocated(By.xpath("//tr[@id='job_" + folderName + "']/td[3]/a/button")));
+        new Actions(getDriver()).moveToElement(projectName).click().perform();
+        getDriver().findElement(By.xpath("//a[@href='/job/ProjectFolder/confirm-rename']")).click();
```
Comment (verafes) [inline src/test/java/school/redrover/CreateFolder1Test.java:59]: try "//a[@href='/job/"+ folderName + "/confirm-rename']"

**Review by verafes — APPROVED** [review]

**Review by Lokankara — APPROVED** [review]

**Pull request merged**